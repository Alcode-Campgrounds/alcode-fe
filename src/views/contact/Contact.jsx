import React from 'react'

export default function Contact() {
    return (
        <div>
            <h2>Drop us a line</h2>
        </div>
    )
}
