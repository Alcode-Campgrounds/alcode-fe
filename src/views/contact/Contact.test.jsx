import React from 'react';
import { render } from '@testing-library/react';
import Contact from './Contact'


it('should render the contact component', () => {
  const { container } = render( <Contact /> );

  expect(container).toMatchSnapshot();
});
