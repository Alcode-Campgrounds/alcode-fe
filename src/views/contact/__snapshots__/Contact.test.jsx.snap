// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the contact component 1`] = `
<div>
  <div
    class="loading-container"
  >
    <h1>
      Contact
    </h1>
  </div>
</div>
`;
